### ISSUE TITLE
[Form] Placeholder text in text input does not have sufficient contrast ratio in Dark Mode

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3025]
### Figma
In Figma placeholder text for TextField and TextArea in Sanoma Learning Dark Mode has contrast ratio of 4.4:1 between text color (#8C8E91) and background color (#25292F).

<img width="1131" height="245" alt="Image" src="https://github.com/user-attachments/assets/b4e4eea4-0a79-4c9d-8a38-21b0a791d786" />

### Storybook
In: 
1. [Examples Form Log In](https://storybook.sanomalearning.design/?path=/story/examples-form--log-in&globals=mode:dark)
2. [Examples Form Composite Form](https://storybook.sanomalearning.design/?path=/story/examples-form--composite-form&globals=mode:dark) 
3. [Form Reset](https://storybook.sanomalearning.design/?path=/story/form-form--reset&globals=mode:dark)
4. [Form Custom Component](https://storybook.sanomalearning.design/?path=/story/form-form--custom-component&globals=mode:dark)
5. [Form All](https://storybook.sanomalearning.design/?path=/story/form-form--all&globals=mode:dark)
6. [Form All Invalid](https://storybook.sanomalearning.design/?path=/story/form-form--all-invalid&globals=mode:dark)
placeholder text for Text Field, Date Field, Time Field and Text Area in Sanoma Learning Dark Mode has contrast ratio of 4.4:1 between text color (#8B8E91) and background color (#252930).

<img width="1038" height="448" alt="Image" src="https://github.com/user-attachments/assets/e7272f55-d82a-4434-ba82-bc541ff4bdb7" />

Proposed solution
For Figma and Storybook contrast ratio between placeholder text and background should be equal to or greater than 4.5:1.
This means changing heading group text color to eg. #8E9194 should fix this issue for background color used in Figma and Storybook Sanoma Learning Dark Mode.

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS team

**Comment by RoaldBoerema** [comment]
Tweaked the token to `rgba( {sanoma-learning.color.palette.grey.900}, 48%)` instead of 47%. Contrast ratio now 4.51:1
